app/controllers/user_credentials_controller.rb

```diff
@@ -14,6 +14,16 @@ def index
     end
   end
 
+  def import
+    file = params[:file]
+    UserCredential.import(file)
```

Comment:
I think we should not rely on the user_id of the JSON here but enforce that the imported credentials belong to the `current_user`, so maybe `current_user.credentials.import(JSON.parse(file))`?